crates/holochain_cascade/src/agent_activity.rs

```diff
@@ -47,7 +47,7 @@ fn merge_hashes(
 
         match valid_activity {
             ChainItems::Full(_) => {
-                // TODO: Currently not handling full headers from
+                // TODO: BACKLOG: Currently not handling full headers from
```

Comment:
I've marked some todos with a sub category of a backlog item because they are very large to actually fix. What should we do with items like this @abe-njama ?

Reply:
@freesig  For those very large fixes, we can have these in the backlog as Tech debt. I can create the backlog items once you provide me with the title and description to go with that.